hack/util.sh

```diff
@@ -196,10 +196,10 @@ function start_os_master {
 
 	echo "[INFO] OpenShift server start at: "
 	date
-	
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 80
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
-	
+
+	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 160
+	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 160
```

Comment:
and not just 40 seconds to start.  40 seconds to **API server** start.  That's really bad.